codecov/__init__.py

```diff
@@ -914,7 +914,7 @@ def main(*argv, **kwargs):
             for path in find_files(sanitize_arg("", codecov.gcov_root or root), "*.gcno", True, dont_search_here):
                 cmd = sanitize_arg("", codecov.gcov_exec or "") + " -pb " + sanitize_arg("", codecov.gcov_args or "") + " " + path
                 write("    Executing gcov (%s)" % cmd)
-                write(try_to_run(cmd))
+                write(try_to_run(cmd, shell=True))
```

Comment:
@nmoinvaz my big worry here is that I think it's possible to insert malicious code here. Am I thinking about this wrong?

Reply:
I think I may have solved the issue another way.. will update PR.

Reply:
Updated.